tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_dialer_dsds_receive_call_on_both_sims.py

```diff
@@ -32,22 +32,16 @@ def test_dsds_receive_call_on_both_sims(self, sim_value, sim_name):
             self.testvars['plivo']['phone_number']
         )
         self.call_uuid = self.plivo.make_call(
-            to_number=self.environment.phone_numbers[sim_value].replace('+', ''))
+            to_number=self.environment.phone_numbers[sim_number - 1].replace('+', ''))
```

Comment:
`phone_number` is a standard python array so we should start at 0. `sim_number` is not decremented, fwiw.